editors/code/package.json

```diff
@@ -210,14 +210,8 @@
     ],
     "configuration": {
       "type": "object",
-      "title": "SourcePawn configuration",
```

Comment:
nit: I prefer that this stays as `SourcePawn`

Reply:
Yes, I just followed this
https://code.visualstudio.com/api/references/contribution-points#:~:text=For%20both%20the,Extension%20Configuration%20Settings%22

Reply:
_"For both the title and displayName fields, words like "Extension", "Configuration", and "Settings" are redundant."_, so it should be `SourcePawn` right?

Reply:
Yes, exactly.

Reply:
Oh no, I read the diff backwards, I thought you added the `configuration` part... Soery about that haha